### PR TITLE
instructor/admin search: results are not highlighted if the search words are in double quotes #4889

### LINKED ISSUE #4889: instructor/admin search: results are not highlighted if the search words are in double quotes
e.g searching for `"Jane Doe"` will not highlight neither `Jane` nor `Doe` in the results

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #4889

### CODE REVIEW

**Comment by tarun14110** [comment]
I had trimmed symbols `".'-` from every word of the search string before highlighting. This approach will highlight the words containing these symbols like `tarun-14110.iiitd`. 
This worked for Instructor Search . But it is not working for admin Search still. The search String is not reaching to admin Search page if someone string is surrounded by ` "`. It is working fine for strings surrounding by other symbols. Do, I need to solve that part within this issue ?
 
Instructor Search page
screenshot from 2016-03-06 12 50 10
screenshot from 2016-03-06 12 52 19


Admin Search page

screenshot from 2016-03-06 13 08 54
screenshot from 2016-03-06 13 09 15

File: src/main/webapp/js/common.js [hunk 1]
```diff
@@ -752,8 +752,15 @@ function sanitizeForJs(string) {
...
     var searchKey = $(searchKeyId).val();
-    var splitSearchKey = searchKey.split(' ');
-    $(sectionToHighlight).highlight(splitSearchKey);
+    // trim symbols around every word in the string
+    var symbolTrimmedSearchKey = [];
+    $.each(searchKey.split(/["'.-]/), function(){
+        symbolTrimmedSearchKey.push($.trim(this));
+    });
+    // remove empty elements from symbolTrimmedSearchKey
+    symbolTrimmedSearchKey=symbolTrimmedSearchKey.filter(function(n){
```
Comment (taniach) [inline src/main/webapp/js/common.js]: Put a space before and after =

**Comment by tarun14110** [comment]
@taniach By mistake I deleted the branch. Restored it. Please review.

**Comment by thyageshm** [comment]
@taniach please merge this one soon

**Pull request merged**